config/setupJest.js

```diff
@@ -190,3 +190,11 @@ jest.mock('react-chartjs-2', () => ({
   Doughnut: () => null,
   Bar: () => null,
 }));
+
+global.fetch = jest.fn(() =>
```

Comment:
We need a mock for `window.fetch` in jest because jest is a node process and window.fetch is undefined.